src/platform-implementation-js/style/gmail.css

```
@@ -429,6 +429,15 @@ div.T-I.inboxsdk__button {
   text-shadow: none;
 }
 
+/* For the outline style button. We need to counteract a few of gmail's styles */
+.ams.inboxsdk__button::before {
+  width: 0;
+  margin: 0;
+}
+.ams.inboxsdk__button:focus {
+  background: rgba(0, 0, 0, 0);
```

To patch up the stylelint error in CI:

```suggestion
}

.ams.inboxsdk__button:focus {
  background: rgb(0 0 0 / 0%);
```